src/content/configuration/index.mdx

```diff
@@ -422,32 +1075,39 @@ module.exports = {
       // must be used on filesystems that doesn't notify on change
       // i. e. nfs shares
     },
-    node: {
-      // Polyfills and mocks to run Node.js-
-      // environment code in non-Node environments.
-      console: false, // boolean | "mock"
-      global: true, // boolean | "mock"
-      process: true, // boolean
-      __filename: "mock", // boolean | "mock"
-      __dirname: "mock", // boolean | "mock"
-      Buffer: true, // boolean | "mock"
-      setImmediate: true // boolean | "mock" | "empty"
+  </advancedCaching>
+  <advancedBuild "#">
+    <default>
+      /* Advanced build configuration (click to show) */
+    </default>
+    infrastructureLogging: {
+      level: "none",
+      level: "error",
+      level: "warn",
+      level: "info", // (default)
+      level: "log",
+      level: "verbose",
+      debug: true,
+      debug: /webpack/,
+      debug: [ "MyPlugin", /webpack/ ]
     },
-    recordsPath: path.resolve(__dirname, "build/records.json"),
-    recordsInputPath: path.resolve(__dirname, "build/records.json"),
-    recordsOutputPath: path.resolve(__dirname, "build/records.json"),
-    // TODO
-  </advanced>
+    parallelism: 1, // number
+    // limit the number of parallel processed modules
+    profile: true, // boolean
+    // capture timing information
+    bail: true, //boolean
+    // fail out on the first error instead of tolerating it.
+    dependencies: ["name"],
+    // When using an array of configs this can be used to reference other
+    // configs and let this config run after the other config at initial build
+  </advancedBuild>
 ```
 
 W> webpack applies configuration defaults after [plugins defaults](/contribute/writing-a-plugin/#configuration-defaults) are applied.
 
 T> Want to rapidly generate webpack configuration file for your project requirements with few clicks away.
```

Comment:
This now looks odd, i will remove the highlights from them
![Screenshot 2020-10-03 221008](https://user-images.githubusercontent.com/10549495/94999895-40df3c00-05c5-11eb-9612-19b6973896c1.png)

When everything is highlighted, user gets confused